en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md

```diff
@@ -3,6 +3,61 @@ This guide provides the recommended strategy for migrating from WSO2 ESB 5.0 to
 
 {!setup/pull/PULL-CONTENT-migration-esb-mi.md!}
 
+-	If you are migrating from a version older than WSO2 ESB 5.0.0, you should first migrate to ESB 5.0.0 and then migrate to the Micro Integrator of EI 7.1.0.
+-	The distribution folder structure has changed from ESB 5.0.0 to EI 7.1:
+	<table>
+		<tr>
+			<th>
+				WSO2 ESB 5.0.0
+			</th>
+			<th>
+				Micro Integrator of EI 7.1.0
+			</th>
+		</tr>
+		<tr>
+			<td>
+				<code><ESB_HOME>/repository/conf</code>
+			</td>
+			<td>
+				<code><EI_HOME>/conf</code>
```

Comment:
Should we call it as `EI_HOME` or `MI_HOME`?

Reply:
changed to MI_HOME